containers/Investment/Redeem/index.tsx

```diff
@@ -60,28 +59,36 @@ class InvestorRedeem extends React.Component<Props, State> {
     const availableTokensOverflow = (new BN(redeemAmount).cmp(new BN(tokenBalance)) > 0);
     const redeemEnabled = redeemLimitSet && !limitOverflow && !availableTokensOverflow;
 
+    const dropAddress = this.props.tinlake.contractAddresses.SENIOR_TOKEN as string;
```

Comment:
here the same: please add this information to the investorView Container and pass it down